app/assets/javascripts/index_map.js

```diff
@@ -45,15 +44,29 @@ function initMap() {
 
   var mc = new MarkerClusterer(map, unit_markers);
 
-  initHeatMap(map, heatMapData);
-
+  initHeatMap(map);
 }
 
-function initHeatMap(map, heatMapData) {
+function initHeatMap(map) {
+  var heatMapPoint;
+  var heatMapData = [];
+  var unit = [];
+  var monthly_rent;
+
+  for (var i in gon.rental_units) {
+    unit[i] = new google.maps.LatLng(gon.rental_units[i].latitude,
```

Comment:
Identifier 'rental_units' is not in camel case.<br>'google' is not defined.<br>'gon' is not defined.